### DISCUSSION TITLE
Dynamic SSG for Docs

### DISCUSSION THREAD

**Discussion by junaid33** [shuding/nextra#740] [Q&A]
I'm trying to create static pages using getStaticPaths and getStaticProps. The pages are being built just fine, but the navigation is not picking up the dynamic pages, and shows [id] in the navigation instead of each path. Is there a way to hide the [id] and get the navigation to use the paths created from getStaticPaths?

**Comment by timvermaercke** [comment]
Can you share how you managed to get your dynamic pages appear in the navigation? Thanks!

**Reply by junaid33** [reply]
I didn't get dynamic SSG navigation working on my docs: https://docs.openship.org. @shuding is this possible with Nextra?

**Comment by thantos** [comment]
Also looking for this, the pages are generated and kind of work, but no navigation or search.

**Comment by crubier** [comment]
Same here, navigation fails

**Comment by moklick** [comment]
It seems that dynamic folders get filtered out https://github.com/shuding/nextra/blob/main/packages/nextra/src/plugin.ts#L87-L89 . It would be great if we could use the pages that get created with `getStaticPaths` somehow. We also couldn't find any workaround for now :/

**Comment by marcklingen** [comment]
I'm also interested in this, I want to generate pages for frontmatter tags that I add to docs. Showing these in the navigation would be super nice.

**Comment by crguezl** [comment]
I wrote this tutorial: 

https://ull-mii-sytws.github.io/temas/web/nextra/server-side-nextra.html#how-server-side-programming-works-in-nextra

contains  an example using both server-side and dynamic routing in Nextra.

Hope it helps

**Reply by mcnojo** [reply]
you are a saint
